tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -250,6 +263,212 @@ function xyz(
 $a#
 );',
             ],
+            'test half-multiline function becomes fully-multiline' => [
+                <<<'EXPECTED'
+<?php
+functionCall(
+    'a',
+    'b',
+    'c'
+);
+EXPECTED
+            ,
+                <<<'INPUT'
+<?php
+functionCall(
+    'a', 'b',
+    'c'
+);
+INPUT
+            ,
+                ['ensure_fully_multiline' => true],
+            ],
+            'test barely multiline function with blank lines becomes fully-multiline' => [
+                <<<'EXPECTED'
+<?php
+functionCall(
+    'a',
+    'b',
+
+    'c'
+);
+EXPECTED
+            ,
+                <<<'INPUT'
+<?php
+functionCall('a', 'b',
+
+    'c');
+INPUT
+            ,
+                ['ensure_fully_multiline' => true],
+            ],
+            'test indentation is preserved' => [
+                <<<'EXPECTED'
+<?php
+if (true) {
+    functionCall(
+        'a',
+        'b',
+        'c'
+    );
+}
+EXPECTED
+            ,
+                <<<'INPUT'
+<?php
+if (true) {
+    functionCall(
+        'a', 'b',
+        'c'
+    );
+}
+INPUT
+            ,
+                ['ensure_fully_multiline' => true],
+            ],
+            'test multiline array arguments do not trigger multiline' => [
+                <<<'EXPECTED'
+<?php
+defraculate(1, array(
+    'a',
+    'b',
+    'c',
+), 42);
```

Comment:
@julienfalque you asked met to add tests with nested arrays, what do you mean exactly? Can you give an example?